markdown/3.x/en/ff-record-list.md

```diff
@@ -145,3 +145,25 @@ We recommend using higher values for `[infinite-debounce-delay]` and `infinite-s
 Please do not use `infinite-scrolling` when the `ff-record-list` is inside a component like `ff-recommendation` or `ff-pushed-products`. The attribute is not designed to work in this context and using it may cause unwanted side effects.
 
 The maximum number of pages that are loaded can be restricted with the `[infinite-max-pages]` attribute.
+
+By default, `ff-record-list` tries to find the closest scrollable parent element and loads new records when the container is scrolled. By setting `infinite-scroll-container` attribute on `ff-record-list`, it is possible to point the container
+directly. Use the appropriate CSS selector as a value of the attribute:
+
+```html
+<div class="scrollable-container">
+    <ff-record-list infinite-scrolling
+                    infinite-debounce-delay="300"
+                    infinite-scroll-margin="-700"
+                    infinite-scroll-container="scrollable-container">
```

Comment:
Shouldn't it be `.scrollable-container`?

Reply:
Indeed. Fixed.